scripts/linters.sh

```diff
@@ -16,11 +16,10 @@
 
```

Comment:
I'm not sure about the value of this script. It could be incorporated straight into the makefile since it's just 3-4 shell commands.

Reply:
I expect this to be expanded over time. We need to call our custom linters here in particular. Before doing so, we need to continue doing a bit of code shuffling. I'd like to move linters to the tools repo and build them into docker images that we can use from other repos.

Reply:
OK. It's just another line of `docker run` if the plan is to package all our linters into a docker container.